libs/api/domains/islykill/src/lib/islykill.module.ts

```diff
@@ -6,15 +6,14 @@ import {
   IslykillApiModuleConfig,
 } from '@island.is/clients/islykill'
 
-import { MainResolver } from './graphql/main.resolver'
 import { IslykillService } from './islykill.service'
 
 @Module({})
 export class IslykillModule {
   static register(config: IslykillApiModuleConfig): DynamicModule {
     return {
       module: IslykillModule,
-      providers: [MainResolver, IslykillService],
+      providers: [IslykillService],
```

Comment:
If this domain doesn’t have a resolver anymore, we should get rid of it and either move the service to the client (if it is useful for “everyone” integrating the client) or just to the UserProfile domain (if it’s mainly useful there).

Reply:
Sounds good. 

I went with placing it with the userprofile. I think it's mainly (only?) going to be used there.